src/pages/Merchant/Register/Register.js

```diff
@@ -0,0 +1,180 @@
+      <div className={styles.registerLeftPanel}>
+        <h1>Buy Best!</h1>
+        <span>100+ products available at best price</span>
+        <img
+          className={styles.registerLeftPanelImage}
+          src="https://image.shutterstock.com/z/stock-vector-set-of-fruit-and-vegetables-logo-for-groceries-agriculture-stores-packaging-and-advertising-318421853.jpg"
+        ></img>
+        <div className={styles.registerLeftPanelIcons}></div>
+      </div>
+      <div className={styles.registerRightPanel}>
+        <form onSubmit={handleSubmit}>
+          <div className={styles["registerRightPanel-mainBody"]}>
+            <h2>Create an account</h2>
+            <div className={styles["registerRightPanel-inputContainer"]}>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  placeholder="Enter first name"
+                  label="First name"
+                  ref={fnameRef}
+                  error={errors?.fname}
+                />
+              </div>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  label="Last name"
+                  placeholder="Enter last name"
+                  ref={lnameRef}
+                  error={errors?.lname}
+                />
+              </div>
+            </div>
+            <div className={styles["registerRightPanel-inputWrapper"]}>
+              <InputControl
+                placeholder="Enter mobile number"
+                label="Mobile number"
+                ref={mobileRef}
+                error={errors?.mobile}
+              />
+            </div>
+            <div className={styles["registerRightPanel-inputWrapper"]}>
+              <InputControl
+                label="Email"
+                placeholder="Enter email"
+                ref={emailRef}
+                error={errors?.email}
+              />
+            </div>
+            <div className={styles["registerRightPanel-inputContainer"]}>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  password="true"
+                  placeholder="Enter password"
+                  label="Password"
+                  ref={passRef}
+                  error={errors?.password}
+                />
+              </div>
+              <div className={styles["registerRightPanel-inputWrapper"]}>
+                <InputControl
+                  placeholder={`Confirm Password`}
+                  label={`Confirm Password`}
+                  ref={confirmpassRef}
+                  error={errors?.confirmpass}
+                  password="true"
+                />
+              </div>
+            </div>
+            <span
+              className={styles["registerRightPanel_helper-text"]}
+              onClick={changeURl}
+            >
+              Already have an account!
+            </span>
```

Comment:
this whole thing should not be clickable instead another text named "login now" should be clickable. and also did you checked that this is working ?